pkg/cmd/factory/factory_impl.go

```diff
@@ -265,3 +276,21 @@ func (f *factoryImpl) findServerAddr() (string, error) {
 	}
 	return "", errors.New("no capture is found")
 }
+
+func checkCDCVersion(client apiv1client.APIV1Interface) error {
+	serverStatus, err := client.Status().Get(cmdconetxt.GetDefaultContext())
+	if err != nil {
+		return errors.Trace(err)
+	}
+	cdcVersion, err := version.GetTiCDCClusterVersion([]string{serverStatus.Version})
+	if err != nil {
+		return errors.Trace(err)
+	}
+	if !cdcVersion.ShouldRunCliWithOpenAPI() {
+		return errors.New("Can’t operate the TiCDC cluster, " +
+			"the cluster version is too old. Only the version of this cluster " +
```

Comment:
Would it be better if we return a error like this?
```go
errors.New("Can’t use this cli-tools to operate the TiCDC cluster, " +
			 "the TiCDC cluster server's version '%s' is older than cli-tools '%s'" +
              "please make sure their versions are matched.", serverVersion, cliVersion)
```

Reply:
the error message is from the feature  spec